atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/ProfilingKeyValueService.java

```diff
@@ -365,6 +365,20 @@ public boolean isInitialized() {
                 logTime("isInitialized"));
     }
 
+    @Override
+    public void compactInternally(TableReference tableRef, boolean inMaintenanceHours) {
+        maybeLog(() -> delegate.compactInternally(tableRef, inMaintenanceHours), (logger, stopwatch) -> {
+            if (inMaintenanceHours) {
+                // Log differently in maintenance hours - if a compactInternally is slow this might be bad in normal
+                // operational hours, but is probably okay in maintenance hours.
+                logger.log("Call to KVS.compactInternally (in maintenance hours) on table {} took {} ms.",
```

Comment:
nit: this section needs to be changed to reference maintenance mode as opposed to maintenance hours